.gitignore

```diff
@@ -6,10 +6,10 @@ composer.lock
 tests/cov/
 
 # Composer binaries
-bin/phpunit
```

Comment:
This prevents that I'm committing these files:
![image](https://user-images.githubusercontent.com/11234139/28614700-05bc8f2a-71f7-11e7-9ac6-fb84b52bf4fe.png)